xterm/index.cgi

```diff
@@ -197,6 +233,7 @@ if (!-r $shellserver_cmd) {
 	}
 defined(getpwnam($user)) || &error(&text('index_euser', &html_escape($user)));
 my $tmpdir = &tempname_dir();
+$ENV{'HISTCONTROL'} = 'ignoredups:ignorespace';
```

Comment:
What does this do?

Reply:
`HISTCONTROL` is evn variable  that controls how commands are saved in the history list.

Reply:
i.e. we send commands though the socket directly, like `'clear'` and `'exit'`, and a few other, so we will use a `space + command`, like `' clear'` or `'  exit'` to make sure it won't be saved to the history list.

Reply:
Ah, thanks!